### ISSUE TITLE
Couldn't regenerate the similar results when conducting the same code

### ISSUE THREAD

**Issue by wangzlgithub** [aristoteleo/dynamo-release#460]
Hi! 
Thanks for this great work! But I met a question.
When  I use the same code to do repetition for my results, I got a really different result and I think this is not the cause of random seed. So why is this happening? The two results is generated within one hour before submitting the issue, and with the same inputs and parameters.
I have pull the codes into github. Please click into it to review.(https://github.com/wangzlgithub/Dynamo_strange_repetition)

**Comment by Xiaojieqiu** [comment]
Thanks for reporting this important issue. We will look into this and get back to you shortly

**Comment by Sichao25** [comment]
Hi, @wangzlgithub. Thank you very much for reporting this issue. Right now we are trying to solve this but we fail to reproduce the same error with our data. So I'd like to ask for more details to better locate the problem. Does this happen only once or does your code always provide different results? If possible, could you provide a sample of your data to help us reproduce the error?

Meanwhile, I also recommend checking our [tutorial](https://dynamo-release.readthedocs.io/en/latest/notebooks/Differential_geometry.html) and playing with different parameters in functions.

**Comment by wangzlgithub** [comment]
Thanks for response. Actually we used dynamo on all data, not subset of it like in notebooks, and got a good result. The notebook I displayed is a trying to explore some interesting things in our results, imitating the part of "scNT-seq neuronal activity" on your tutorial. I'm so sorry that I can't provide samples from our unpubublished data. Right now it still has these strange results in that notebook multiple times. Maybe it occured cause of our data?

Finally, thank you again for developing dynamo. Maybe l'll come back to you to explore this question after the paper is published.
